labs/a4.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Lab 4 - Post Install
+layout: lab
+---
+
+# Generating SSH Keys
+Since passwords can be guessed, a secure SSH setup will usually use key-based authentication instead of (or in addition to) a passphrase. You may already have an ssh key, so run the following command to see if you have one:
+
+(note: We won’t give instructions here for Windows machines. To follow these instructions, SSH into `ssh.ocf.berkeley.edu` with your OCF account’s credentials and continue from there)
+
+```
+ls .ssh
+```
+
+If you see id_rsa or id_edsca listed, you already have an SSH key. If you don’t, run this command to generate one:
```

Comment:
It might be worth noting that the * in `id_*` is the algorithm used to generate the ssh key. This might also be a good question for the google form.